### DISCUSSION TITLE
How to run migration ?

### DISCUSSION THREAD

**Discussion by shethinfocon** [PiranhaCMS/piranha.core#1988] [General]
Hi,

Can anyone please guide how to add new database migration for MSSQL ?

I am getting error message "More than one DbContext was found. Specify which one to use. Use the '-Context' parameter for PowerShell commands and the '--context' parameter for dotnet commands."

**Comment by tidyui** [comment]
Could you provide more information. Have you added your own DbContext for you custom entities to your Piranha application and want to add migrations for it? If so, just like the error message says you need to specify the class name of your DbContext as the application already contains other DbContext and the command line tool doesn't know which one to choose.

`dotnet migrations add MigrationName --context MyDbContext`

Here is the official documentation from Microsoft (https://learn.microsoft.com/en-us/ef/core/managing-schemas/migrations/providers?tabs=dotnet-core-cli)

Best regards
